lib/prefix.h

```diff
@@ -335,6 +335,17 @@ union prefixconstptr {
 	prefixtype(prefixconstptr, const struct prefix_rd,   rd)
 } TRANSPARENT_UNION;
 
+#ifdef __INTELLISENSE__
+/*
+ * Fix Visual Studio Code error: argument of type "struct prefix *" is
```

Comment:
This one is specific to `lib/prefix.h`, but for consistency maybe we should move `TRANSPARENT_UNION` to `lib/compiler.h` so everything is in one place.  Especially since `pimd/pim_addr.h` uses it too.